indigo_app/views/works.py

```diff
@@ -176,7 +176,25 @@ def get_success_url(self):
         return reverse('work', kwargs={'frbr_uri': self.work.frbr_uri})
 
 
-class UnapproveWorkView(WorkViewBase, View):
```

Comment:
this isn't right; ApproveWorkView has fallen away and UnapproveWorkView is now its own thing: see https://github.com/laws-africa/indigo/pull/1951/commits/125dd5c4f893abcd83a04a3b021dc4cd4ef864d0

Reply:
I think it's fixed in https://github.com/laws-africa/indigo/pull/1965